.circleci/config.yml

```diff
@@ -17,6 +17,7 @@ jobs:
       ruby_version: "3.2"
     environment:
       FRONTEND: "/home/circleci/project/spec/solidus_frontend_app_template.rb"
+      AUTHENTICATION: "devise"
```

Comment:
Did we use to test this extension with solidus_auth_devise?

Reply:
Yes, it is necessary for triggering saving a customer on stripe within some specs. I had to force it because solidus main wasn't picking it up anymore (probably related to solidus_frontend support being removed).